Makefile

```diff
@@ -185,23 +230,23 @@ $(LOCALBIN):
 	mkdir -p $(LOCALBIN)
 
 ## Tool Binaries
+KUBECTL ?= kubectl
 KUSTOMIZE ?= $(LOCALBIN)/kustomize
 CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
 ENVTEST ?= $(LOCALBIN)/setup-envtest
 
 ## Tool Versions
-KUSTOMIZE_VERSION ?= v5.0.1
-CONTROLLER_TOOLS_VERSION ?= v0.11.3
+KUSTOMIZE_VERSION ?= v5.2.1
+CONTROLLER_TOOLS_VERSION ?= v0.13.0
```

Comment:
I think we should upgrade to the latest `v0.15.0`, also `v0.14.0` works (and then re-run `make build`) otherwise we hit this issue:

```bash
bin/controller-gen rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xa0de6f]

goroutine 116 [running]:
go/types.(*Checker).handleBailout(0xc0002d3200, 0xc0020e1d40)
	/usr/lib/golang/src/go/types/check.go:367 +0x88
panic({0xbc7860?, 0x12b48a0?})
	/usr/lib/golang/src/runtime/panic.go:770 +0x132
go/types.(*StdSizes).Sizeof(0x0, {0xdc3678, 0x12bd080})
	/usr/lib/golang/src/go/types/sizes.go:228 +0x30f
go/types.(*Config).sizeof(...)
	/usr/lib/golang/src/go/types/sizes.go:333
go/types.representableConst.func1({0xdc3678?, 0x12bd080?})
	/usr/lib/golang/src/go/types/const.go:76 +0x9e
go/types.representableConst({0xdc9a50, 0x1287f80}, 0xc0002d3200, 0x12bd080, 0xc0020e14b0)
	/usr/lib/golang/src/go/types/const.go:92 +0x192
go/types.(*Checker).representation(0xc0002d3200, 0xc0020c1d00, 0x12bd080)
	/usr/lib/golang/src/go/types/const.go:256 +0x65
go/types.(*Checker).implicitTypeAndValue(0xc0002d3200, 0xc0020c1d00, {0xdc36a0, 0xc00020ebd0})
	/usr/lib/golang/src/go/types/expr.go:375 +0x2d7
go/types.(*Checker).assignment(0xc0002d3200, 0xc0020c1d00, {0xdc36a0, 0xc00020ebd0}, {0xc95293, 0x14})
	/usr/lib/golang/src/go/types/assignments.go:52 +0x2e5
go/types.(*Checker).initConst(0xc0002d3200, 0xc00206e5a0, 0xc0020c1d00)
	/usr/lib/golang/src/go/types/assignments.go:126 +0x2c5
go/types.(*Checker).constDecl(0xc0002d3200, 0xc00206e5a0, {0xdc62c0, 0xc001baa020}, {0xdc62c0, 0xc001baa040}, 0x0)
	/usr/lib/golang/src/go/types/decl.go:490 +0x311
go/types.(*Checker).objDecl(0xc0002d3200, {0xdcf1e0, 0xc00206e5a0}, 0x0)
	/usr/lib/golang/src/go/types/decl.go:191 +0xa49
go/types.(*Checker).packageObjects(0xc0002d3200)
	/usr/lib/golang/src/go/types/resolver.go:693 +0x4dd
go/types.(*Checker).checkFiles(0xc0002d3200, {0xc0018b6030, 0x5, 0x5})
	/usr/lib/golang/src/go/types/check.go:408 +0x1a5
```

```suggestion
CONTROLLER_TOOLS_VERSION ?= v0.15.0
```

Reply:
yes agreed I kept whatever was generated by operator-sdk but the newer the better

Reply:
done